components/tree/private/branch.jsx

```diff
@@ -155,13 +240,20 @@ const renderBranch = (children, props) => {
 		</div>
 	);
 
-	// TODO: Remove tabbing from anchor tag AND button / add tabIndex={-1} when keyboard navigation is present.
 	return (
 		<li
 			id={props.htmlId}
 			role="treeitem"
 			aria-level={props.level}
 			aria-expanded={isExpanded ? 'true' : 'false'}
+			aria-label={props.node.nodes && props.node.nodes.length > 0 ? props.node.label : null}
+			tabIndex="0"
```

Comment:
All items need to be `tabIndex="-1"` except for the active/focused one, so that there's only one tab stop